apps/ryal_core/config/dev.exs

```diff
@@ -1 +1,3 @@
 use Mix.Config
+
+import_config "../test/support/dummy/config/config.exs"
```

Comment:
Kinda. It means that when I run the Ryal application in Dev (`iex -S mix`) I can load up the Dummy application.